crates/bevy_pbr/src/render_graph/pbr_pipeline/pbr.frag

```diff
@@ -310,7 +380,9 @@ void main() {
     vec3 diffuse_ambient = EnvBRDFApprox(diffuseColor, 1.0, NdotV);
     vec3 specular_ambient = EnvBRDFApprox(F0, perceptual_roughness, NdotV);
 
-    output_color.rgb = light_accum + (diffuse_ambient + specular_ambient) * AmbientColor;
+    output_color.rgb = light_accum;
```

Comment:
Tiny comments here would make it easier for folks new to rendering to figure out exactly what's happening here.

See the `// tone_mapping` comment just below.